pup-0003.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Cherry-Picking Commits
+author: David Davis
+created: 01-Apr-2017
+status:
+---
+
+## Summary
+
+Currently in Pulp, we merge features to master. However, hotfixes and bug fixes
+are merged to release branches which then are merged to newer release branches
```

Comment:
A few accuracy things about this section/line.
1) s/release/devel/
2) Only one merge forward occurs but this section says two. The bug fix is merged to the devel branch which are then merged to master.
3) hotfixes are pretty special so perhaps disinclude them.